<launch>
  <!-- Output -->
  <arg name="output/clusters" default="output/clusters"/>
  <arg name="input/cluster0" default="input/cluster0"/>
  <arg name="input/cluster1" default="input/cluster1"/>
  <arg name="output_frame_id" default="base_link"/>
  <!-- Node -->
  <node pkg="cluster_merger" exec="cluster_merger_node" name="cluster_merger" output="screen">
    <remap from="~/output/clusters" to="$(var output/clusters)"/>
    <remap from="input/cluster0" to="$(var input/cluster0)"/>
    <remap from="input/cluster1" to="$(var input/cluster1)"/>
    <param name="output_frame_id" value="$(var output_frame_id)"/>
  </node>
</launch>
